systemtests/tests/gfapi-fd/testrunner

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+set -e
+set -u
+set -x
+#
+# Run a simple backup
+#   then restore it.
+#
+
+
+
+
+TestName="$(basename "$(pwd)")"
+export TestName
+
+JobName=backup-bareos-fd
+
+#shellcheck source=../environment.in
+. ./environment
+
+umount_glusterfs(){
+#  make sure umount is successful
+tries=60
+echo "umounting glusterfs..."
+while sudo mount | grep "${tmp}/data" >/dev/null 2>&1; do
```

Comment:
invoking sudo here will not only require sudo to be installed, but also require sudo to be setup in a way where the user running the test can execute these commands without intervention.
I'm not 100% sure, but I believe this will hang indefinitely waiting for a password on a default installation.